code/obj/machinery/door/airlock.dm

```diff
@@ -282,18 +285,22 @@ Airlock index -> wire color are { 9, 4, 6, 7, 5, 8, 1, 2, 3 }.
 	visible = 0
 
 /obj/machinery/door/airlock/glass/command
+		name = "command airlock"
 		icon = 'icons/obj/doors/Doorcom-glass.dmi'
 		req_access = list(access_heads)
 
 /obj/machinery/door/airlock/glass/engineering
+		name = "command airlock"
 		icon = 'icons/obj/doors/Dooreng-glass.dmi'
 		req_access = list(access_engineering)
 
 /obj/machinery/door/airlock/glass/medical
+		name = "medical airlock"
 		icon = 'icons/obj/doors/Doormed-glass.dmi'
 		req_access = list(access_medical)
 
 /obj/machinery/door/airlock/classic
+	name = "classic airlock"
```

Comment:
not a fan of 'classic'

Reply:
I'm trying "large" since that's the only natural way I can think to describe a non-perpective door